website/project/model.py

```diff
@@ -2119,7 +2119,11 @@ def fork_node(self, auth, title='Fork of '):
                 if forked_node is not None:
                     forked.nodes.append(forked_node)
 
-        forked.title = title + forked.title
+        if title == 'Fork of ' or title == '':
```

Comment:
Yes. The method is written to prepend whatever title is supplied to the beginning of the original node title.  When no title is supplied, 'Fork of ' is added to the original node title.  If the method is forking your components, an empty string is added to the original node title.